lib/aullar/spec/styling_spec.moon

```diff
@@ -329,6 +329,49 @@ describe 'Styling', ->
           2, 's1:s3', 3,
         }, styling\get(1, 6)
 
+  describe 'get_nearest_style_marker(pos)', ->
+    it 'generally works', ->
```

Comment:
Instead of "generally works", try to come up with something describing what it actually does when it works. E.g. does it return the nearest preceding marker, or the nearest subsequent marker?